README.md

```diff
@@ -1 +1,103 @@
-# Hello-microverse
\ No newline at end of file
+<div align="center">
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Awesome books ](#-awesome-books-)
+  - [🛠️ Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Hello-microverse <a name="about-project"></a>
+
+**Hello-microverse**, in this project, i setup only Hello-microverse.
+
+## 🛠️ Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+HTML | CSS 
+
+### Key Features <a name="key-features"></a>
+
+- **[key_feature_1]**
+- **[key_feature_2]**
+- **[key_feature_3]**
+
```

Comment:
Great job adding the README file, but I think it will be better if you specify the key features of your project: e.g
Html project that displays Hello Microverse 😊 🟢